### ISSUE TITLE
Attribute `log_dir` not accessible from trainer.

### ISSUE THREAD

**Issue by jzazo** [PyTorchLightning/pytorch-lightning#4892]
## 🐛 Bug

Attribute `log_dir` not accessible from trainer: `trainer.log_dir`
[Documentation](https://pytorch-lightning.readthedocs.io/en/latest/trainer.html#log-dir) indicates this should be accessible.

### To Reproduce

Call `trainer.log_dir`.

### Expected behavior
Should return current experiment `log_dir`.

**Comment by rohitgr7** [comment]
`log_dir` is not available on the 1.0.8 release. You are looking at the [latest docs](https://pytorch-lightning.readthedocs.io/en/latest/trainer.html#log-dir) and not the [stable ones](https://pytorch-lightning.readthedocs.io/en/stable/trainer.html#log-dir).

They are available on master. You can install it using:
```
pip install git+https://github.com/PyTorchLightning/pytorch-lightning.git -U
```

**Comment by jzazo** [comment]
Ok, thank you!
